conf/attribute-resolver.xml.tmpl

```diff
@@ -381,7 +396,7 @@
               var ncrs = sln_courses.getValues().size();
               for (var i=0; i<ncrs; i++) {
                  var crs = sln_courses.getValues().get(i);
-                 if (crs.startsWith("course")) {
+                 if (crs.startsWith("course") && crs.length()>15) {
```

Comment:
Maybe a comment re: derivation of these magic numbers? (15 here and 49 on line 409 and 46 on line 540)